include/oc_core_res.h

```diff
@@ -64,6 +64,14 @@ typedef struct oc_device_info_t
   oc_string_t dmv;                       ///< data model version
   oc_core_add_device_cb_t add_device_cb; ///< callback when device is changed
   void *data;                            ///< user data
+
+#ifdef OC_HAS_FEATURE_BRIDGE
+  oc_string_t
+    ecoversion; ///< Version of ecosystem that a Bridged Device belongs to.
```

Comment:
Remove. I don't see this used anywhere in this PR, add it when its functionality is added.

Reply:
fixed